webpack/components/Content/Details/TableSchema.js

```diff
@@ -4,7 +4,7 @@ import { translate as __ } from 'foremanReact/common/I18n';
 import {
```

Comment:
I think `RepositoriesTableSchema` or even the very verbose `ContentDetailsRepositoriesTableSchema` would be a better name for this file so its more clear where it fits in

Reply:
Renamed to ContentDetailRepositoryTableSchema.